lua/wire/client/text_editor/texteditor.lua

```diff
@@ -619,10 +620,11 @@ function EDITOR:Paint()
 		self.Caret = self:CursorToCaret()
 	end
 
-	surface_SetDrawColor(0, 0, 0, 255)
+	local backgroundColor = self:GetSyntaxColor("background") or Color(32, 32, 32)
```

Comment:
I'd prefer not having a call to ``Color`` every render. You can properly store the default background color with self.Colors: https://github.com/wiremod/wire/blob/e115f4bec552792871aa32fe96fc3939731c41dd/lua/wire/client/text_editor/texteditor.lua#L87-L89